TestApp/Source/Android/app/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -61,34 +61,36 @@ Java_com_android_appviewer_AndroidViewAppActivity_finishEngine(JNIEnv* env, jobj
 {
 }
 
-void AndroidLogMessage(const char* message)
+void LogMessage(const char* message, babylon::LogLevel level)
 {
-    __android_log_write(ANDROID_LOG_INFO, "BabylonNative", message);
-}
-
-void AndroidWarnMessage(const char* message)
-{
-    __android_log_write(ANDROID_LOG_WARN, "BabylonNative", message);
-}
-
-void AndroidErrorMessage(const char* message)
-{
-    __android_log_write(ANDROID_LOG_ERROR, "BabylonNative", message);
+    switch (level)
+    {
+    case babylon::LogLevel::Log:
+        __android_log_write(ANDROID_LOG_INFO, "BabylonNative", message);
+        break;
+    case babylon::LogLevel::Warn:
+        __android_log_write(ANDROID_LOG_WARN, "BabylonNative", message);
+        break;
+    case babylon::LogLevel::Error:
+        __android_log_write(ANDROID_LOG_ERROR, "BabylonNative", message);
+        break;
+    }
 }
 
 JNIEXPORT void JNICALL
 Java_com_android_appviewer_AndroidViewAppActivity_surfaceCreated(JNIEnv* env, jobject obj, jobject surface)
 {
     if (!runtime)
     {
-        // register console outputs
-        babylon::Runtime::RegisterLogOutput(AndroidLogMessage);
-        babylon::Runtime::RegisterWarnOutput(AndroidWarnMessage);
-        babylon::Runtime::RegisterErrorOutput(AndroidErrorMessage);
-
         ANativeWindow *window = ANativeWindow_fromSurface(env, surface);
 
-        runtime = std::make_unique<babylon::RuntimeAndroid>(window, "file:///data/local/tmp");
+        runtime = std::make_unique<babylon::RuntimeAndroid>(
+            window, 
+            "file:///data/local/tmp",
+            [](const char* message, babylon::LogLevel level)
```

Comment:
Can you put `LogMessage` directly here without the lambda? I think it will auto convert when there is no capture.